nixos/modules/services/web-apps/tachidesk-server.nix

```
@@ -0,0 +1,235 @@
+{ config, pkgs, lib, ... }:
+
+let
+  cfg = config.services.tachidesk-server;
+  inherit (lib) mkOption mdDoc mkEnableOption mkIf types;
+in
+{
+  options = {
+    services.tachidesk-server = {
+      enable = mkEnableOption (mdDoc "A free and open source manga reader server that runs extensions built for Tachiyomi.");
```

`mkEnableOption` automatically prepends "Whether to enable", a capitalized `A` would seem a bit out of place.

```suggestion
      enable = mkEnableOption (mdDoc "Tachidesk, a free and open source manga reader server that runs extensions built for Tachiyomi.");
```

---

Done